### PR TITLE
change CAS login verbiage

### PR DESCRIPTION
**Pull request by groundedgopher**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ ssl_cert_path: /etc/pki/tls/certs
 ssl_key_path: /etc/pki/tls/private
 
 # UI
-cas_screen_welcome_instructions: Enter your Username and Password
-cas_screen_welcome_security: For security reasons, please Log Out and Exit your web browser when you are done accessing services that require authentication!
+cas_screen_welcome_instructions: Login with your OUNetID (4x4)
```
Comment (jsnshrmn) [inline defaults/main.yml]: No need to change the default values.  These message defaults are set to the cas out of the box defaults.  We can just set the values appropriately in production. If you have bits of text you want to set, define a var with the original value here, and then we'll set the vars on deploy.

File: templates/messages.properties.j2 [hunk 2]
```diff
@@ -22,7 +22,7 @@
 screen.welcome.welcome=Congratulations on bringing CAS online!  To learn how to authenticate, please review the default authentication handler configuration.
 screen.welcome.security={{ cas_screen_welcome_security }}
 screen.welcome.instructions={{ cas_screen_welcome_instructions }}
-screen.welcome.label.netid=<span class="accesskey">U</span>sername:
+screen.welcome.label.netid=<span class="accesskey">O</span>UNetID (4x4):
```
Comment (jsnshrmn) [inline templates/messages.properties.j2]: don't hardcode values here, create a var that gets templated in, like cas_screen_welcome_instructions.  Note the naming convention of vars.

File: defaults/main.yml (hunk 1 above)
Comment (groundedgopher) [reply defaults/main.yml]: I have set the variables in vagrant infrastructure in the host_vars now. In place of the hard coded version is the original default values.

File: templates/messages.properties.j2 (hunk 2 above)
Comment (groundedgopher) [reply templates/messages.properties.j2]: This is now a variable {{ cas_screen_label_netid }}. It is also in the main.yml file in vagrant_infrastructure

File: defaults/main.yml (hunk 1 above)
Comment (groundedgopher) [reply defaults/main.yml]: There is a pull request for these changes in vagrant_infrastructure also

**Review by jsnshrmn — APPROVED** [review]
Looks great, will need to resolve merge conflicts, but that's not your problem.

**Pull request merged**